config.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file is not used?

Reply:
Ops created it for testing. Can I add it to ignore instead?